tests/dogtag/pytest-ansible/pytest/ca/bugzilla/test_bug_1787115_Copy_SKI_from_CSR_to_Certificate_Signed.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/python
+# -*- coding: UTF-8 -*-
+
+"""
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Description: Bug - 1787115 Copy SKI from CSR to Certificate Signed.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Author: Gaurav Swami <gswami@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2020 Red Hat, Inc. All rights reserved.
+#
+#   This copyrighted material is made available to anyone wishing
+#   to use, modify, copy, or redistribute it subject to the terms
+#   and conditions of the GNU General Public License version 2.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE. See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public
+#   License along with this program; if not, write to the Free
+#   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+#   Boston, MA 02110-1301, USA.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+
+import os
+import sys
+import pytest
+
+import re
+import tempfile
+
+from pki.testlib.common.utils import ProfileOperations, log
+
+try:
+	from pki.testlib.common import constants
+except Exception as e:
+	if os.path.isfile('/tmp/test_dir/constants.py'):
+		sys.path.append('/tmp/test_dir/')
+		import constants
+
+profile = 'caCACert'
+profop = ProfileOperations(nssdb=constants.NSSDB)
+
+
+def test_profile_update_SKI_parameters(ansible_module):
+	"""
+		:id: 8f761dd1-219e-404a-a1ea-333cf2a606b9
+		:Title: Bug 1787115 - SubjectAltNameExtInput does not display text fields to the enrollment page
+		:Description: Bug 1787115 - SubjectAltNameExtInput does not display text fields to the enrollment page
+		:Requirement:
+		:CaseComponent: \-
+		:Setup:
+			1. Use the subsystems setup in ansible to run subsystem commands
+
+		:Steps:
+			1. Disable profile caCACert.
+			2. Edit caCACert profile to add parameter 'policyset.caCertSet.8.default.params.useSKIFromCertRequest=true'
+			3. Enable profile.
+
+		:ExpectedResults:
+			1. profile should be disabled.
+			2. caCACert profile must include parameter 'policyset.caCertSet.8.default.params.useSKIFromCertRequest=true'
+			3. profile should be enabled.
+
+	"""
```

Comment:
I added, on the runner:

```python
    ansible_module.ping()
```

And it worked. :o 

To me, this says that there's a race: `py.test` isn't loading `ansible` modules fast enough and so `ansible_module.pki` gets called before it is loaded.

Reply:
PR fixing the issue: https://github.com/ansible/pytest-ansible/pull/50